idle-batch/src/main/kotlin/com/swm/idle/batch/crawler/WorknetPostParser.kt

```
@@ -0,0 +1,145 @@
+        }.getOrElse { e->
+            errorRecorder.recordError(con.location)
+            throw e
+        }
+    }
+
+    private fun extractAddress(driver: WebDriver,vararg cons: CrawlerConsts): String {
+        runCatching {
+            for (con in cons) {
+                val address = driver.findElement(By.xpath(con.value)).text
+                return address.replace("지도보기", "").trim().replace(Regex("\\(\\d{5}\\)"), "").trim()
+            }
```

detekt (1.23.7)

[warning] This loop contains an unconditional jump expression which essentially renders it useless as it will exit the loop during the first iteration.

(detekt.potential-bugs.UnconditionalJumpStatementInLoop)